### DISCUSSION TITLE
git-committers2: Change Text between Logo and Avatars

### DISCUSSION THREAD

**Discussion by christophhillisch** [squidfunk/mkdocs-material#6548] [Ideas]
Hi,

we are using the [git-committers2](https://github.com/ojacques/mkdocs-git-committers-plugin-2) to display the contributors at the bottom of the page. To me the text "GitHub" between the logo of GitHub and the avatars of the contributors is not ideal, since our readers do not really care about the fact whether we use GitLab or GitHub. I would rather use this text to explain to naive users that these avatars to the right are the contributors. Would it be possible to make this text configurable?

To make it very clear what I am describing, here the screenshot of what I mean:
![contributors](https://github.com/squidfunk/mkdocs-material/assets/44038602/478d0c4f-5055-42a9-814c-c9eb907f7d38)
In this I would like to have "Contributors" spelled instead of "GitHub"

**Comment by kamilkrzyskow** [comment]
Hello @christophhillisch,
leaving the configurability part aside, to achieve that currently you can override the template partial responsible for the the rendering (partials/source-file.html).
https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme
https://github.com/squidfunk/mkdocs-material/blob/175df613970869744114e0c4299f0e448aba8737/src/templates/partials/source-file.html#L92-L96

Another solution is to add extra JavaScript that finds the appropriate element on the page and replaces the text and removes the icon.

**Reply by christophhillisch** [reply]
Thanks for that suggestion! I exactly implemented it as you said.
